interview-questions.md

```diff
@@ -7,60 +7,74 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is git? What is the difference between git and Github?
 
-  Your answer:
-
-  Researched answer:
-
+  Your answer: 
+    Git is a command prompt in terminal that interacts with data stored on Github. Github is a site that securely stores repositories and their branches so multiple can work on the same sections of code at the same time. 
 
+  Researched answer: 
+    Git is a command-line tool software installed locallay on the system that is used as a version control system to manage code history and code sharing. GitHub is a GUI service hosted on the web that is used to securely store Git repositories and centralized code. 
 
 2. What types of evaluations will return a Boolean value?
 
-  Your answer:
+  Your answer: 
+    Greater than ">", less than "<", stricly equals "===", and loose equals "==", bang operators used alongside strict equality "!==", greaterthan or equal to ">=" , and less than or equal to "<=". 
 
-  Researched answer:
+  Researched answer: 
+    Conditional statements. 
 
 
 
 3. What is an index? What is the difference between index and value?
 
-  Your answer:
+  Your answer: 
+    An index is the location of of a value stored on an array. A value is the data stored on each index of an array. 
 
-  Researched answer:
+  Researched answer: 
+    An index is a particular location of a value in an array. A value is a javascript data type stored in each element of an array. 
 
 
 
 4. What is a built-in method?
 
-  Your answer:
+  Your answer: 
+    Built-in methods are functions that are built-in to the javascript ptogramming language. They are commonly used methods that the designers of javascript were gracious enough to include into the language as shortcuts for commonly used methods.
 
-  Researched answer:
+  Researched answer: 
+    Pre-made javascript functionality that can be used on string data types.
```

Comment:
Okay this is a good example of where you would be served by diving super deep. Built in methods are available to almost all data types and they have specific uses in each case.